app/services/idv/aamva_state_maintenance_window.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Idv
+  class AamvaStateMaintenanceWindow
+    class << self
+      def in_maintenance_window?(state)
+        return false unless (window = window_for_state(state))
+
+        Time.use_zone(window['tz']) do
+          start_time = Time.parse("today #{window['start_time']}")
+          end_time = Time.parse("today #{window['end_time']}")
+          Time.zone.now > start_time && Time.zone.now < end_time
```

Comment:
I would also inject `now = Time.zone.now` as an optional param for easier testing maybe
```suggestion
          (start_time...end_time).cover?(Time.zone.now)
```

Reply:
Nice! I wanted something like `.cover?` but didn't know it existed.